src/org/jitsi/impl/neomedia/VideoMediaStreamImpl.java

```diff
@@ -440,10 +440,15 @@ else if(closestAspect == null)
         = new RTCPReceiverFeedbackTermination(this);
 
     /**
+     *
+     */
+    private final PaddingTermination paddingTermination = new PaddingTermination();
+
+    /**
      * The <tt>RemoteBitrateEstimator</tt> which computes bitrate estimates for
      * the incoming RTP streams.
      */
-    private final RemoteBitrateEstimator remoteBitrateEstimator
+    private final RemoteBitrateEstimatorSingleStream remoteBitrateEstimator
```

Comment:
I needed the implementation because it implements the `TransformEngine` interface and now I hook the RemoteBitrateEstimator instance into the transform chain. I can make the `RemoteBitrateEstimator` interface extend the `TransformEngine` instead.